code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -1684,3 +1705,22 @@
 	name = "smoke grenade shell"
 	nade_type = /obj/item/explosive/grenade/smokebomb
 	icon_state = "smoke_shell"
+
+
+/datum/ammo/bullet/shrapnel/flechette
+	name = "fragmentation flechette"
+	icon_state = "flechette"
+	flags_ammo_behavior = AMMO_SHRAPNEL
+
+/datum/ammo/bullet/shrapnel/flechette/New()
+	. = ..()
+	accuracy = CONFIG_GET(number/combat_define/max_hit_accuracy)
+	accurate_range = CONFIG_GET(number/combat_define/min_shell_range)
+	accuracy_var_low = CONFIG_GET(number/combat_define/med_proj_variance)
+	accuracy_var_high = CONFIG_GET(number/combat_define/med_proj_variance)
+	max_range = CONFIG_GET(number/combat_define/close_shell_range)
+	damage = CONFIG_GET(number/combat_define/low_hit_damage)
+	damage_var_low = -CONFIG_GET(number/combat_define/low_proj_variance)
+	damage_var_high = CONFIG_GET(number/combat_define/low_proj_variance)
+	damage_falloff *= 0.5
```

Comment:
Use a proper config here, and `=` instead of `*=`

Reply:
this is the same for all damage_falloff defines